digitalpy/core/zmanager/integration_manager.py

```diff
@@ -11,33 +11,50 @@
 import zmq
 
 class IntegrationManager:
+    def __init__(self, integration_manager_puller_protocol: str, integration_manager_puller_address: str, integration_manager_puller_port: int,
+                 integration_manager_publisher_protocol: str, integration_manager_publisher_address: str, integration_manager_publisher_port: int) -> None:
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WPS319 Found bracket in wrong position